src/lib/config_parser.c

```diff
@@ -112,3 +112,101 @@ bool config_parse(const char *filename, parse_config_fn_t parse_config_fn, void
 exit:
 	return ret;
 }
+
```

Comment:
I will sync up with you on this. What I tried to do is create a 3rd-party_config.c where this file is read into a list of struct repo. My initial understanding was bundle-add will build on the same idea of adding bundles to those structs.

keeping @otaviobp loop.